packages/block-editor/src/components/inspector-controls-child/index.native.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { Children } from '@wordpress/element';
+import { createSlotFill, BottomSheetConsumer } from '@wordpress/components';
+
+const { Fill, Slot } = createSlotFill( 'InspectorControlsChild' );
+
+const FillWithSettingsButton = ( { children, button, showSheet } ) => {
+	return (
+		<>
+			<Fill>
+				{ showSheet && (
```

Comment:
Is `showSheet` intended to avoid rendering unused, invisible UI? Does it make sense to avoid rendering the entire `Fill`? It may have no impact at all, but I just noticed that [`InspectorControls` avoid rendering `Fill`](https://github.com/WordPress/gutenberg/blob/16d13227b38eb818c4e133ee456fb7d0fe3bb45e/packages/block-editor/src/components/inspector-controls/index.native.js#L22-L24).

Reply:
Good Call!